src/create_shards.c

```diff
@@ -422,16 +428,43 @@ ParseWorkerNodeFile(char *workerNodeFilename)
 			continue;
 		}
 
-		/* parse out the node name and node port */
-		parsedValues = sscanf(workerNodeLine, workerLinePattern, nodeName, &nodePort);
-		if (parsedValues != 2)
+		/* parse line; node name is required, but port is optional */
+		fieldCount = sscanf(workerNodeLine, workerLinePattern, nodeName, nodePortString);
```

Comment:
`sscanf` returns EOF on failure. Since EOF is -1, I can see that we catch the failures below. But, should we add a comment that we already check the failure, or would it be useless?